utils/update_checkout/update_checkout/update_checkout.py

```diff
@@ -94,6 +94,10 @@ def update_single_repository(args):
         with shell.pushd(repo_path, dry_run=False, echo=False):
             cross_repo = False
             checkout_target = None
+
+            if should_fetch:
+                shell.run(['git', 'fetch'], echo=True)
```

Comment:
I think some extra arguments are necessary:
```suggestion
                shell.run(['git', 'fetch'], echo=True)
                shell.run(["git", "fetch", "--recurse-submodules=yes", "--tags"],
                          echo=True)
```

Reply:
Actually, this exact `git fetch` command is executed [a few lines later in the file](https://github.com/apple/swift/blob/85bfbe2dd33f7fea023f8d2a88689810b870adbd/utils/update_checkout/update_checkout/update_checkout.py#L129):

```python
            if checkout_target:
                shell.run(['git', 'status', '--porcelain', '-uno'],
                          echo=False)
                shell.run(['git', 'checkout', checkout_target], echo=True)

            # It's important that we checkout, fetch, and rebase, in order.
            # .git/FETCH_HEAD updates the not-for-merge attributes based on
            # which branch was checked out during the fetch.
            shell.run(["git", "fetch", "--recurse-submodules=yes", "--tags"],
                      echo=True)
```

But maybe it's being run too late?